### PR TITLE
#5 TodosSerializer implemented + tested for json.

### LINKED ISSUE #5: Output in file todos.json
The output should be written to a file called ``todos.json``, to be created in the same folder where the utilitary is run.
If the todos.json file already exists from a previous run, it should be removed and a new one should be created.

### PR DESCRIPTION
**Pull request by criske**
closes #5

### CODE REVIEW

**Comment by charlesmike** [comment]
@criske thank you for your Pull Request. I'll assign someone to review it soon.

File: src/main/java/com/selfxdsd/todocli/JsonTodosSerializer.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    @Override
+    public URI serialize() {
+        final URI location;
+        try (final JsonWriter writer = Json
+            .createWriterFactory(Map.of(JsonGenerator.PRETTY_PRINTING, true))
+            .createWriter(new FileWriter(this.getFile(true)))) {
+            final JsonArrayBuilder array = Json.createArrayBuilder();
+            for (final Todo todo : todos) {
+                final JsonObject object = Json.createObjectBuilder()
+                    .add("ticket", todo.getTicketID())
```
Comment (criske) [inline src/main/java/com/selfxdsd/todocli/JsonTodosSerializer.java:74]: @amihaiemil I'll update with proper abstraction when #4 is resolved.

**Comment by charlesmike** [comment]
@amihaiemil I couldn't find any assignee for this task. This is either because there are no contributors with role ``REV`` available or because the project does not have enough funds.

Please, make sure there is at least one available contributor with the required role and the project can afford to pay them.

File: src/main/java/com/selfxdsd/todocli/JsonTodosSerializer.java [hunk 2]
```diff
@@ -99,12 +99,12 @@ public URI serialize() {
     private File getFile(final boolean deleteFirst) throws URISyntaxException,
         IOException {
         //folder where application is running.
-        final String parent = JsonTodosSerializer.class
+        final String parent = new File(JsonTodosSerializer.class
```
Comment (criske) [inline src/main/java/com/selfxdsd/todocli/JsonTodosSerializer.java:102]: @amihaiemil todos.json is now created in the same folder of the app jar (tested manually).

**Comment by amihaiemil** [comment]
@rultor merge it please

**Pull request merged**